demomultistoreform/src/Form/ContentBlockFormDataHandler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\Module\DemoMultistoreForm\Form;
+
+use PrestaShop\Module\DemoMultistoreForm\Entity\ContentBlock;
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler\FormDataHandlerInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use PrestaShopBundle\Entity\Shop;
+
+class ContentBlockFormDataHandler implements FormDataHandlerInterface
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $em;
```

Comment:
Nice-to-have: don't use 2-letter variable names. It can confuse junior software engineers 😄  .